.github/workflows/docker-backend.yml

```diff
@@ -30,9 +30,15 @@ env:
 jobs:
   docker:
     runs-on: ubuntu-latest-16core
+    # If workflow is running on a branch ref, use the nightly snapshot nexus repo. Otherwise, use the releases or oss repo depending on the value of the `oss` input.
     env:
-      IMAGE_PATH: "${{ inputs.registry }}/${{ inputs.name }}"
+      REGISTRY: "${{ github.ref_type == 'branch' && 'docker-nightly.nexus.famedly.de' || ( inputs.oss && 'docker-oss.nexus.famedly.de' || 'docker-releases.nexus.famedly.de' ) }}"
```

Comment:
Not sure if this is correct/sufficient for the future. We might have nightly/weekly tags that are snapshots, too.
It should work if you use the registry in the existing bash script ifs respectively.

Reply:
In GH actions you can't do $IMAGE_PATH: "$REGISTRY/$IMAGE_NAME", github does not expand the existing variables.

See f.x. https://stackoverflow.com/questions/62716758/how-to-extend-environment-variables-in-github-actions-with-existing-ones